src/modules/market/components/market-positions-list--position/market-positions-list--position.jsx

```diff
@@ -11,14 +11,24 @@ import MarketOutcomeTradingIndicator from "modules/market/containers/market-outc
 export default class MarketPositionsListPosition extends Component {
   static propTypes = {
     outcomeName: PropTypes.string.isRequired,
-    position: PropTypes.object.isRequired,
-    openOrders: PropTypes.array.isRequired,
+    position: PropTypes.shape({
+      netPosition: PropTypes.object,
+      qtyShares: PropTypes.object,
+      purchasePrice: PropTypes.object,
+      unrealizedNet: PropTypes.object,
+      realizedNet: PropTypes.object,
+      totalNet: PropTypes.object
+    }).isRequired,
     isExtendedDisplay: PropTypes.bool.isRequired,
     isMobile: PropTypes.bool.isRequired,
-    outcome: PropTypes.object,
+    outcome: PropTypes.object.isRequired,
```

Comment:
`outcome` is undefined when used in `market-positions-list.jsx` so it can't be isRequired